pages/agent/index.tsx

```diff
@@ -0,0 +1,238 @@
+          </h1>
+          <p className="fr-text--lead">
+            Accessible à toutes les administrations, collectivités et services
+            publics de l’Etat.
+          </p>
+        </header>
+        <ButtonAgentConnect />
+      </div>
+      <img src="/images/lp-agent/secure-folder 1.svg" alt="" />
+    </section>
+    <section className="fr-grid-row fr-grid-row--gutters value">
+      <h2 className="fr-sr-only">Quels avantages pour les agents ?</h2>
+      <div className="fr-col-12  fr-col-md-4">
+        <h3>Un accès simple et rapide</h3>
+        <img src="/images/lp-agent/padlock-security 1.svg" alt="" />
+        <p>
+          Le bouton AgentConnect vous permet d’accéder facilement à l’espace
+          agent grâce à votre adresse email professionnelle.
+        </p>
+      </div>
+      <div className="fr-col-12  fr-col-md-4">
+        <h3>Encore plus de données</h3>
+        <img src="/images/lp-agent/data-classification 1.svg" alt="" />
+
+        <p>
+          En tant qu’agent public, accédez automatiquement aux informations
+          protégées des entreprises (non diffusibles, statuts, actes, bilans
+          financiers).
+        </p>
+      </div>
+      <div className="fr-col-12  fr-col-md-4">
+        <h3>Une équipe à votre écoute</h3>
+        <img src="/images/lp-agent/string-phone-communication 1.svg" alt="" />
+        <p>
+          Vous rêvez d’une nouvelle donnée, d’une fonctionnalité, vous
+          rencontrez un problème ? Vos retours sont très précieux pour continuer
+          d’améliorer cet espace.
+        </p>
+      </div>
+    </section>
+    <section>
+      <h2>Comment l’Annuaire vous aide au quotidien ?</h2>
+      <div
+        className="fr-grid-row fr-grid-row--gutters"
+        style={{ marginTop: '1rem' }}
+      >
+        <CaseExample
+          title="Vérifiez facilement l’éligibilité des entreprises en consultant leur bilan financier et leurs statuts."
+          description="Pour instruire vos demandes de subventions"
+          image="/images/lp-agent/woman-work-with-dashboard 1.svg"
+        />
+        <CaseExample
+          title="Accédez directement aux informations des non diffusibles, sans avoir à leur demander de documents en plus."
+          description="Pour aider les entreprises individuelles"
+          image="/images/lp-agent/Group.svg"
+        />
+      </div>
+    </section>
+    <Container
+      style={{
+        background: 'var(--background-alt-blue-france)',
+      }}
+    >
+      <section
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          gap: '1rem',
+        }}
+      >
+        <h2>Rejoignez les agents qui utilisent déjà l’espace agent !</h2>
+        <ButtonAgentConnect />
+      </section>
+    </Container>
+    <section>
+      <h3>
+        L’Annuaire des Entreprises est opéré par la DINUM, avec le partenariat
+        de l’INSEE, l’INPI, l’ADEME etc.
+      </h3>
```

Comment:
Why not a `h2` ?

Reply:
- The figma maquette was suggesting a h3
- The structuration of information still makes sense that way (and I though that reducing number of h2 reduce the time for scanning the page with a screen reader)

```
+ Rejoignez les agents qui utilisent déjà l’espace agent
---+ L’Annuaire des Entreprises est opéré par la DINUM, avec le partenariat de l’INSEE, l’INPI, l’ADEME etc.
```

Not quite sure about the content though «  de l’INSEE, l’INPI, l’ADEME etc » sounds like we are tired of listing all the partners :p. 

Maybe change to something more neutral : 
 L’Annuaire des Entreprises est opéré par la DINUM, en partenariat avec de nombreuses administrations : »

Reply:
Pushed a proposal for new wording.

Reply:
I second this